### PR TITLE
Login page Alert adding done

### PR DESCRIPTION
**Pull request by Hemant2335**
Done adding Alerts on Login page .

Attachments

Screenshot (70)
Screenshot (72)
Screenshot (71)

### CODE REVIEW

File: pages/login.tsx [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { LoginWithGooglePopUp, auth } from "@/firebase";
 import { useAuthState } from "react-firebase-hooks/auth";
 
 import { DarkModeToggle } from "../Components/components";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
```
Comment (lalitmee) [inline pages/login.tsx:11]: I think this should not be imported here. It should go inside `_app.tsx`. As we will be using this `toast` throughout the whole application so importing it again and again will not be the right approach.

File: pages/login.tsx [hunk 2]
```diff
@@ -18,6 +20,33 @@ const Login = () => {
...
+  const success = () => {
+    toast.success("Sucessfully Login", {
+      position: "bottom-right",
+      autoClose: 5000,
+      hideProgressBar: false,
+      closeOnClick: true,
+      pauseOnHover: true,
+      draggable: true,
+      progress: undefined,
+      theme: "light",
```
Comment (lalitmee) [inline pages/login.tsx:33]: you don't need to provide these props every time if you move `ToastContainer` in the `page` where `<App />` is there. And put this `ToastContainer` there. Please read the documentation of `react-toastify` properly and then make the changes

**Review by lalitmee — CHANGES_REQUESTED** [review]

File: pages/login.tsx (hunk 2 above)
Comment (Hemant2335) [reply pages/login.tsx:33]: Thanks for your suggestions I'll be doing the required changes

**Comment by Hemant2335** [comment]
Hloo could you please help me out with this issue

ss
Screenshot (118)

**Comment by lalitmee** [comment]
> Hloo could you please help me out with this issue
> 
> ss Screenshot (118)

This is due to some firebase config. @PiyushKalyanpy, I think we have it mentioned in the docs?

**Comment by PiyushKalyanpy** [comment]
The requested changes have not been implemented, so we are closing this pull request. Please feel free to resubmit the changes when they are ready.

**Pull request closed without merging**